auth/session_test.go

```diff
@@ -16,6 +16,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/stretchr/testify/assert"
```

Comment:
group (and alphabetise) imports - your IDE or `goimports` should be able to do this automatically

```suggestion
	"github.com/couchbase/sync_gateway/base"
	"github.com/stretchr/testify/require"
	"github.com/stretchr/testify/assert"
```